src/util/chat.hpp

```diff
@@ -280,3 +280,14 @@ namespace big::chat
 				});
 	}
 }
+
+namespace big
+{
+	struct chat_message
+	{
+		std::string sender;
+		std::string content;
+	};
+
+	extern std::queue<chat_message> translate_Queue;
```

Comment:
2 things I'd change here:
1. Change `extern` to `inline` as there's no relevant `chat.cpp` file so the instantiation origin is lost and requires further searching through the code base.
2. Odd naming, it's using a mix between camelCase and snake_case. Please swap this to snake_case.